app/models/classes/format_url.rb

```diff
@@ -0,0 +1,18 @@
+class Classes::FormatUrl
```

Comment:
models/classes - what do you mean by that namespacing? 

I think your file structure looks strange. Would I be right in thinking this takes a string, and turns it into a clickable link if it is an address? 

If so, it looks like business logic, or even a helper method. It should not have anything to do with models. Maybe it could go in PostHelpers?